public_html/assets/css/nf-core.css

```diff
@@ -946,3 +946,9 @@ Based on https://codepen.io/wintr/pen/beBJBb */
 #json_schema {
   font-size: 0.7rem;
 }
+.schema_row input[type=number]{
+  width: 4rem;
+}
```

Comment:
Why do we want narrower numeric input fields?

Reply:
It looks weird to me if the up-down buttons are so far away:
<img width="488" alt="Screenshot 2020-03-23 12 09 12" src="https://user-images.githubusercontent.com/6169021/77310836-506cf680-6cff-11ea-8bfb-d70c9a7124a7.png">

Reply:
...I guess. Looks weird to me if the inputs aren't all the same width though:

![image](https://user-images.githubusercontent.com/465550/77313172-a3e14380-6d03-11ea-85cc-624ccf9056fa.png)

Personally I prefer it the way it was - I don't really like having a chunk of space that is wasted, even if the buttons are kind of distant. I'm not sure that people will use the buttons much anyway, I never do.

Reply:
how about putting it on the left side:
<img width="244" alt="Screenshot 2020-03-23 13 48 57" src="https://user-images.githubusercontent.com/6169021/77318305-28848f80-6d0d-11ea-8d8f-e357f811d0c3.png">

Reply:
I still prefer it as it was to be honest, but if you have a strong attachment to this then that's fine 😄

Reply:
Not a hill I have to die on.